README.md

```diff
@@ -372,6 +372,26 @@ cp sample-dcrdata.conf ~/.dcrdata/dcrdata.conf
 Then edit dcrdata.conf with your dcrd RPC settings. See the output of `dcrdata
 --help` for a list of all options and their default values.
 
+### Note: Edit postgresql settings in drcdata.conf file. 
+```ini
+pg=true 
+pgdbname=<<DB_NAME>> e.g dcrdata_mainnet
+pguser=<<DB_OWNER>> e.g dcrdata
+pgpass=<<PASSWORD>>
+pghost=<<PG_HOST>> e.g 127.0.0.1:5432
+```
+
+#### (Optional): Disable the TLS connection with dcrd
```

Comment:
I'd like to remove this section since it's generally advised against.  An op can figure it out if needed.

Reply:
done